### PR TITLE
PDF watermark added

### PR DESCRIPTION
**Pull request by muddhit**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #219 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [x] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Python/PDFWatermark/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Amazon Price Tracker 
+- This script built in Python is an Amazon Price Tracker.
```
Comment (tejan-singh) [inline Python/PDFWatermark/README.md]: Please check the document before sending PR.

**Review by tejan-singh — CHANGES_REQUESTED** [review]

**Comment by muddhit** [comment]
Apologies for that @tejan-singh

**Comment by muddhit** [comment]
@yashaswibiyahut  @tejan-singh @SANKET7738 done with required changes

**Review by tejan-singh — CHANGES_REQUESTED** [review]
The generated pdf is corrupted. Please check your code again for correct working.

File: Python/PDFWatermark/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## PDF Watermark script
+- This script built in Python is used to add PDF WATERMARK . 
+- The user enters :
+    - The Original PDF Name.
```
Comment (tejan-singh) [inline Python/PDFWatermark/README.md]: Specify that the input pdf needs to be in the current location

**Review by tejan-singh — CHANGES_REQUESTED** [review]

**Comment by muddhit** [comment]
> The generated pdf is corrupted. Please check your code again for correct working.

It is working fine in my system and i have also got the output,should i share the image of the merged output pdf

**Comment by tejan-singh** [comment]
You can add a sample pdf for testing purpose in your PR.

**Comment by muddhit** [comment]
> You can add a sample pdf for testing purpose in your PR.

You mean the output pdf which is watermarked?

**Comment by tejan-singh** [comment]
You can to add an input pdf for testing

**Comment by muddhit** [comment]
Added.

**Pull request merged**